setup.cfg

```diff
@@ -1,21 +1,23 @@
-[bumpversion]
-current_version = 0.2.8
-commit = True
-tag = True
-
-[bumpversion:file:setup.py]
-search = version='{current_version}'
-replace = version='{new_version}'
-
-[bumpversion:file:huvr_client/__version__.py]
-search = __version__ = '{current_version}'
-replace = __version__ = '{new_version}'
-
-[bdist_wheel]
-universal = 1
-
-[flake8]
-exclude = docs
-
-[aliases]
-# Define setup.py command aliases here
+; TODO - add this back ...
+;
+;[bumpversion]
+;current_version = 0.2.8
+;commit = True
+;tag = True
```

Comment:
i think won't use the version from python files, but rely on github actions and releases - if possible